spec.html

```diff
@@ -3179,7 +4004,7 @@ <h1>IsStrictReference ( _V_ )</h1>
         <h1>HasPrimitiveBase ( _V_ )</h1>
         <emu-alg>
           1. Assert: Type(_V_) is Reference.
-          1. If Type(_V_'s base value component) is Boolean, String, Symbol, or Number, return *true*; otherwise return *false*.
+          1. If Type(_V_'s base value component) is Boolean, String, Symbol, BigInt or Number, return *true*; otherwise return *false*.
```

Comment:
```suggestion
          1. If Type(_V_'s base value component) is Boolean, String, Symbol, BigInt, or Number, return *true*; otherwise return *false*.
```